src/appkit.cs

```diff
@@ -25489,11 +25498,43 @@ interface NSAccessibilityTable : NSAccessibilityGroup {
 	}
 
 	[Mac (10,10)]
+	[Protocol]
 	interface NSAccessibilityOutline : NSAccessibilityTable {
+		[Abstract]
+		[NullAllowed, Export ("accessibilityLabel")]
+		new string AccessibilityLabel { get; }
```

Comment:
Why re-define it (and others) ?

This is an interface so it inherits the same member from `NSAccessibilityTable`, so the contract remains the same.

Reply:
Removing, if contract is the same.